### ISSUE TITLE
Make and populate pages for data catalog metadata in Wagtail

### ISSUE THREAD

**Issue by jenniferthibault** [18F/fec-cms#1401]
Links to this content in Adv data tabs created in https://github.com/18F/fec-cms/issues/1400

_______________________________
## Design template from https://github.com/18F/openFEC-web-app/issues/2318

![screen shot 2017-10-26 at 6 51 38 pm](https://user-images.githubusercontent.com/11636908/32080620-bec0dfc4-ba7e-11e7-8d9b-2e3f7940adcb.png)

## Content
📝  [Google Doc of metadata content](https://docs.google.com/a/gsa.gov/document/d/15ELHgzOIdS-x3oD3pByuhhzhSTgf6xo_boNECsvo_zI/edit?usp=sharing)

**Completion criteria**
- [x] Populate text for each page (identified below)
- [ ] Complete first level review of content
- [x] Decide on a place where these pages should live in the Wagtail folder structure. (should be the same place as in https://github.com/18F/fec-cms/issues/1399, depending on which issue gets picked up first)
- Use the Custom page wagtail template with paragraph and table block modules to create metadata pages for:
  - [x]  Lobbyist bundled contributions file metadata
  - [x]  Independent expenditures file metadata
  - [x]  Electioneering communications file metadata
  - [x]  Communication costs file metadata
  - [x]  Candidate summary file metadata
  - [x]  New statements of candidacy file metadata
  - [x]  Committee summary file metadata
  - [x]  Leadership PACs and sponsors metadata
  - [x]  Lobbyist/registrant committees file metadata
  - [x]  New committee registrations file metadata
  ~- [ ]  Senate unofficial electronic filings file metadata~ no download file for this
  - [x]  Unverified filers file metadata

**Comment by PaulClark2** [comment]
@llienfec   these pages are ready for your review

Administrative fines file description
Lobbyist bundled contributions file description
Candidate summary file description
Committee summary file description
Communication costs file description
Electioneering communications file description
Independent expenditures file description
Leadership PACs and sponsors description
Lobbyist/registrant committees file description
New committee registrations file description
New statements of candidacy file description
Unverified filers file description

**Comment by llienfec** [comment]
3 parts to my work on this:
- Reorganize/edit/standardize language
- Update document that tracks what language changes are made and why
- Update each wagtail page with new language **and** creating new data column with example data in the HTML table

This list is just to help me track progress on these pages: 
- [ ] Administrative fines file description
- [ ] Lobbyist bundled contributions file description
- [ ] Candidate summary file description
- [ ] Committee summary file description
- [ ] Communication costs file description
- [ ] Electioneering communications file description
- [ ] Independent expenditures file description
- [ ] Leadership PACs and sponsors description
- [ ] Lobbyist/registrant committees file description
- [ ] New committee registrations file description
- [ ] New statements of candidacy file description
- [ ] Unverified filers file description

**Comment by llienfec** [comment]
I have drafted language updates for the first 5 in the list above.  Will work on the next five before entering everything into Wagtail.  Entering updates last to make sure that as much of the information and structure is standardized across pages as possible.

**Comment by AmyKort** [comment]
Thank you! I know this has been a really heavy lift.

**Comment by llienfec** [comment]
Finished drafting language for these pages.  Still need to enter changes into Wagtail.

:lock: https://docs.google.com/document/d/1se3cmFyeiUi9B6ybhGZ6qrtn_NwhwBmHlG6EDnS4KMo/edit?usp=sharing :lock:

**Comment by llienfec** [comment]
Entered pages in Wagtail.  Notes about strange cases:
- AF page was already created and linked to the AF page.  I deleted Paul's page and updated the language on the original page.  It's saved as a draft.  **Note:** The AF page has a data link that stops at 2017, so we need to figure out how to link it to the more updated data once it's live.
- Committee summary page - fixed a bug in the HTML that was keeping at least 20 rows from showing up in the chart.  
- Electioneering communications - on classic, only year available is 2017-18.  There's language about other years in the explanation.  I kept it in for now, but if we're only offering the current cycle, then Paul will need to delete this text.
- General question re: describing the 18 digit image #.  For the source description part, will efilings always be "90"?  The description "9x" is confusing.

Pages are all updated in Wagtail and ready for @PaulClark2's review.  Track changes is in the doc posted above.
